ontology/cdno.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: ontology_detail
+id: cdno
+title: Crop Dietary Nutrition Ontology
+jobs:
+  - id: https://travis-ci.org/Southern-Cross-Plant-Science/cdno
+    type: travis-ci
+build:
+  checkout: git clone https://github.com/Southern-Cross-Plant-Science/cdno.git
+  system: git
+  path: "."
+contact:
+  email: l.andres.hernandez.18@student.scu.edu.au
+  label: Liliana Andres Hernandez
+  github: 
+description: CDNO provides structured terminologies to describe nutritional attributes of crop plants and their harvested materials that contribute to human diet. 
+domain: dietary nutritional components, nutritional component concentrations
+homepage: https://github.com/Southern-Cross-Plant-Science/cdno
+products:
+  - id: cdno.owl
+    name: "Crop Dietary Nutrition Ontology main release in OWL format"
+  - id: cdno.obo
+    name: "Crop Dietary Nutrition Ontology additional release in OBO format"
+dependencies:
+- id: ro
+- id: bfo
+- id: pato
+- id: chebi
+- id: po
+- id: envo
+
+tracker: https://github.com/Southern-Cross-Plant-Science/cdno/issues
+license:
+  url: https://creativecommons.org/licenses/by/1.0/
+  label: CC-BY
+activity_status: active
+---
```

Comment:
Quality control suggested the following errors in this file:

```
FAILURES:
ERROR:ontology/cdno.md: 15:10: trailing spaces (trailing-spaces)
ERROR:ontology/cdno.md: 16:163: trailing spaces (trailing-spaces)
ERROR:ontology/cdno.md: 25:1: wrong indentation: expected 2 but found 0 (indentation)
```

Please fix them all first!

Reply:
Thanks, sorry. I will find and remove the spaces.